### PR TITLE
Adding internal port information for a resource

### PR DESCRIPTION
**Pull request by abhinitmodi**

### CODE REVIEW

File: Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1 [hunk 1]
```diff
@@ -138,6 +138,9 @@ function Get-Resources
                 {
                     $property = New-Object Microsoft.VisualStudio.Services.DevTestLabs.Model.PropertyBagData($false, $winRmHttpPortMap[$resource.Name])
                     $propertyBag.Add("WinRM_Http", $property)
+
+					$property = New-Object Microsoft.VisualStudio.Services.DevTestLabs.Model.PropertyBagData($false, "5985")
```
Comment (pankajkhanzode) [inline Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1]: extra space before $property - can be removed

Comment (NikhilRanjanKumar) [inline Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1]: Fix the indentation

File: Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1 [hunk 2]
```diff
@@ -145,6 +148,9 @@ function Get-Resources
                 {
                     $property = New-Object Microsoft.VisualStudio.Services.DevTestLabs.Model.PropertyBagData($false, $winRmHttpsPortMap[$resource.Name])
                     $propertyBag.Add("WinRM_Https", $property)
+
+					$property = New-Object Microsoft.VisualStudio.Services.DevTestLabs.Model.PropertyBagData($false, "5986")
```
Comment (NikhilRanjanKumar) [inline Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1]: Fix the indentation

**Comment by NikhilRanjanKumar** [comment]
Signed off

File: Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1 (hunk 2 above)
Comment (pankajkhanzode) [inline Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1]: same here

**Comment by pankajkhanzode** [comment]
Signed off

**Comment by kmkumaran** [comment]
Old PR. Closing it.

**Pull request closed without merging**